crates/dbs-virtio-devices/src/fs/device.rs

```diff
@@ -295,10 +289,10 @@ impl<AS: GuestAddressSpace> VirtioFs<AS> {
                         "bootstrap_path": "{}",
                         "blob_cache_dir": "{}"
                     }}"#,
-                    cfg, source, &blob_config.work_dir
+                    cfg, source, blob_config.as_ref().unwrap().work_dir
```

Comment:
Need to return Error here? This is the behavior at runtime.

Reply:
FsError::InvalidData

Reply:
fixed all the `unwrap()` with `ok_or()` to define `FsError`